### PR TITLE
[All Hosts] (browsers) small fixes

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Review changes from other PRs that came in after the PRs were approved/merged.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c4bf8c0](https://github.com/OfficeDev/office-js-docs-pr/commits/c4bf8c0cf8e7285c49a6693347844d78c79c14c1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2219) |
[docs/testing/sideload-office-add-ins-for-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/testing/sideload-office-add-ins-for-testing.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing?branch=pr-en-us-2219) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009241910359276-2219%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DfKIdBY5yIZnf09UzBS%2F9oyEg51z%2FgFMgJdY03%2Bjt60%3D&st=2020-09-24T19%3A06%3A19Z&se=2020-10-25T19%3A11%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/testing/sideload-office-add-ins-for-testing.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following video walks you through the process of sideloading your add-in in
 6. Verify that your add-in is installed. For example, if it is an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in, the pane should appear.
 
 > [!NOTE]
-> To test your Office Add-in with Microsoft Edge with the original WebView (EdgeHTML), an additional configuration step is required. In a Windows Command Prompt, run the following line: `npx office-addin-dev-settings appcontainer EdgeWebView --loopback --yes`. This is not required when Office is using the Chromium-based Edge WebView2. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/sideload-office-add-ins-for-testing.md:40]: You want to keep this version of the note, don't you? It seems like this is reverting...

Comment (Rick-Kirkham) [reply docs/testing/sideload-office-add-ins-for-testing.md:40]: I'll be danged if I know why that happened. That change was merged to master and I pulled from master before I created this branch. At any rate, I'll fix and push. Thanks.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[897d46a](https://github.com/OfficeDev/office-js-docs-pr/commits/897d46a502eef78be88268f0bd5ed7d74587a32d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2219) |
[docs/testing/sideload-office-add-ins-for-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/testing/sideload-office-add-ins-for-testing.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing?branch=pr-en-us-2219) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009242040437280-2219%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=k%2BrEPjeulZ%2BpcN5aUjQ61ZdbvbIo86Zt%2FVybHIEyDnw%3D&st=2020-09-24T20%3A36%3A21Z&se=2020-10-25T20%3A41%3A21Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/testing/sideload-office-add-ins-for-testing.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ The following video walks you through the process of sideloading your add-in in
 6. Verify that your add-in is installed. For example, if it is an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in, the pane should appear.
 
 > [!NOTE]
-> To test your Office Add-in with Microsoft Edge with the original WebView (EdgeHTML), an additional configuration step is required. In a Windows Command Prompt, run the following line: `npx office-addin-dev-settings appcontainer EdgeWebView --loopback --yes`. This is not required when Office is using the Chromium-based Edge WebView2. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
+> To test your Office Add-in with Microsoft Edge with the original WebView (EdgeHTML), an additional configuration step is required. In a Windows Command Prompt, run the following line: `npx office-addin-dev-settings appcontainer EdgeWebView --loopback --yes`. This is not required when Office is using the Chromium-based Edge WebView2. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/sideload-office-add-ins-for-testing.md:40]:
```suggestion
> To test your Office Add-in with Microsoft Edge with the original WebView (EdgeHTML), an additional configuration step is required. In a Windows Command Prompt, run the following line: `npx office-addin-dev-settings appcontainer EdgeWebView --loopback --yes`. This is not required when Office is using the Chromium-based Edge WebView2. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Just need to fix link.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf7d6a6](https://github.com/OfficeDev/office-js-docs-pr/commits/bf7d6a62ba170804e8b26a601612217366297d25)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins?branch=pr-en-us-2219) |
[docs/testing/sideload-office-add-ins-for-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/browsers-edge/docs/testing/sideload-office-add-ins-for-testing.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-office-add-ins-for-testing?branch=pr-en-us-2219) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009242048024811-2219%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=am7Dn6FFC6LUh6xjGxKG9jTRRHjkf270BkOqM3QBFeM%3D&st=2020-09-24T20%3A43%3A37Z&se=2020-10-25T20%3A48%3A37Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**